src/DynamoCoreWpf/Views/Preview/WatchTree.xaml

```diff
@@ -77,12 +77,34 @@
     <Grid>
         <TreeView Name="treeView1"
                   ItemsSource="{Binding Children}"
-                  Background="Transparent"
+                  Background="White"
```

Comment:
Fixes this bug:
![image](https://cloud.githubusercontent.com/assets/8158404/12576448/613a845e-c41c-11e5-96da-08c40f0a97b6.png)